examples/stackflow-spa/src/seed-design/ui/help-bubble.tsx

```
@@ -16,69 +17,79 @@ export interface HelpBubbleTriggerProps extends Omit<SeedHelpBubble.RootProps, "
-    );
-  },
-);
-
-export interface HelpBubbleAnchorProps extends Omit<SeedHelpBubble.RootProps, "children"> {
+export const HelpBubbleTrigger = forwardRef<
+  HTMLButtonElement,
+  HelpBubbleTriggerProps
+>((props, ref) => {
+  const {
+    showCloseButton = false,
+    title,
```

Biome (2.1.2)

[error] This variable title is unused.

Unused variables are often the result of an incomplete refactoring, typos, or other sources of bugs.
You can use the ignoreRestSiblings option to ignore unused variables in an object destructuring with a spread.

(lint/correctness/noUnusedVariables)